runtime/mount/endpoints/run.py

```diff
@@ -164,10 +179,21 @@ async def run(s: Span, ctx: Context) -> HandlerResult:
                 k_proc.proc.send_signal(signal=signal.SIGINT)
                 continue
 
+            if msg["type"] == "run_cell" and not is_session_owner:
+                await ctx.send_message(
+                    orjson.dumps(
+                        {
+                            "type": "error",
```

Comment:
maybe we send some sort of custom error here?

also should we block other actions too?